examples/python/documentation/resources.py

```diff
@@ -125,9 +125,9 @@ def __init__(self, scope: Construct, name: str):
                 vpc_id="vpcs",
                 egress=[
                     {
-                        "fromPort": 0,
-                        "toPort": 0,
-                        "ciderBlocks": ["0.0.0.0/0"],
+                        "from_port": 0,
+                        "to_port": 0,
+                        "cider_blocks": ["0.0.0.0/0"],
```

Comment:
Spotted another typo that was there before
```suggestion
                        "cidr_blocks": ["0.0.0.0/0"],
```

Reply:
<img src="https://media1.giphy.com/media/ZcIN7ufn6xXaAPmN57/giphy.gif"/>